FestivalPlanner/films/views.py

```diff
@@ -790,16 +805,17 @@ def get_reviewer_row(self, reviewer):
         }
         return reviewer_row
 
-    def get_fan_judgements(self, reviewer):
+    def _get_fan_judgements(self, reviewer):
         fan_judgements = []
         dropdown_rows = []
         for fan in self.fan_list:
+            filter_kwargs = {'film__in': self.reviewed_films, 'film__reviewer': reviewer, 'film_fan': fan}
             ratings = (FilmFanFilmRating.film_ratings
-                       .filter(film__reviewer=reviewer, film_fan=fan)
+                       .filter(**filter_kwargs)
                        .select_related('film'))
             rating_set = set([rating.film for rating in ratings])
             votes = (FilmFanFilmVote.film_votes
-                     .filter(film__reviewer=reviewer, film_fan=fan)
+                     .filter(**filter_kwargs)
                      .select_related('film'))
             vote_set = set([vote.film for vote in votes])
             judge_set = rating_set & vote_set
```

Comment:
Please shorten the code in this `for` loop.

Reply:
Shortened it by moving the initialization of the judged films set per fan to a new method.

The loop is still a bit long, but I think it's useful for understanding when we leave it like that.